task/runner.go

```diff
@@ -542,20 +596,22 @@ func humanizeCatalystError(err error) error {
 		}
 	}
 	if strings.Contains(errMsg, "error running ffprobe") && strings.Contains(errMsg, "exit status 1") {
-		return errInvalidVideo 
+		return errInvalidVideo
 	}
 	if strings.Contains(errMsg, "failed probe/open") {
 		return errProbe
 	}
 
 	// Livepeer pipeline errors
 	if strings.Contains(errMsg, "unsupported input pixel format") {
-		return errors.New("unsupported input pixel format, must be 'yuv420p' or 'yuvj420p'")
-	} else if strings.Contains(errMsg, "Unsupported video input") {
-		// TODO(yondonfu): This check probably does not work because the error message will be lowercased
-		return errors.New("unsupported file format")
+		return UnretriableError{errors.New("unsupported input pixel format, must be 'yuv420p' or 'yuvj420p'")}
+	} else if strings.Contains(errMsg, "unsupported video input") {
+		return UnretriableError{errors.New("unsupported file format")}
```

Comment:
Need this as part of the mc errors as well? 
https://github.com/livepeer/task-runner/pull/158/files

Reply:
Ah yeah! Removed this check as it's already caught on that list now